client-spark/spark-3-shaded/src/main/resources/META-INF/LICENSE

```diff
@@ -225,31 +225,25 @@ io.netty:netty-codec-stomp
 io.netty:netty-codec-xml
 io.netty:netty-common
 io.netty:netty-handler
+io.netty:netty-transport-native-unix-common
 io.netty:netty-handler-proxy
 io.netty:netty-resolver
 io.netty:netty-resolver-dns
-io.netty:netty-resolver-dns-classes-macos
-io.netty:netty-resolver-dns-native-macos
 io.netty:netty-transport
+io.netty:netty-transport-rxtx
+io.netty:netty-transport-sctp
+io.netty:netty-transport-udt
 io.netty:netty-transport-classes-epoll
 io.netty:netty-transport-classes-kqueue
+io.netty:netty-resolver-dns-classes-macos
```

Comment:
Should not be included because it is excluded at the final shaded jar.

Reply:
removed